lang/main.json

```diff
@@ -403,7 +403,8 @@
         "toggleFilmstrip": "Show or hide video thumbnails",
         "toggleScreensharing": "Switch between camera and screen sharing",
         "toggleShortcuts": "Show or hide keyboard shortcuts",
-        "videoMute": "Start or stop your camera"
+        "videoMute": "Start or stop your camera",
+		"videoquality": "Manage call quality"
```

Comment:
Based on the casing of "videoMute", this key should probably be camel case as well.